### PR TITLE
Moresimonfixes

### PR DESCRIPTION
**Pull request by sbrugel**
various fixes to different things, including
- fix for autopaging of the leaderboard if user's rank is 10, 20, 30, or any multiple of 10
- removed the random apostrophe at the end of private thread names
- don't add admins that aren't course staff into PQs (because no one likes random pings!)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,6 @@
 		"@typescript-eslint/eslint-plugin": "^4.23.0",
 		"@typescript-eslint/parser": "^4.23.0",
 		"eslint": "^7.26.0",
-		"tsc-watch": "^4.2.9",
-		"typescript": "^4.1.3"
```
Comment (joshualyon7) [inline package.json:61]: What happened here?

Comment (sbrugel) [reply package.json:61]: this was a long while ago so i don't exactly remember, but i may have been messing around with a later version of djs. i'll get rid of this change soon

Comment (bladetyr) [inline package.json:61]: I'd say put the typescript version number back (just to make sure a later version won't mess with anything). TS is on 4.6.2 as of right now, so I don't want anything to accidentally break

**Review by bladetyr — CHANGES_REQUESTED** [review]

**Comment by joshualyon7** [comment]
Merge into dev 😡

**Comment by sbrugel** [comment]
damn it i keep forgetting to do that 🤦 bear with

**Review by joshualyon7 — APPROVED** [review]
nice

**Review by joshualyon7 — CHANGES_REQUESTED** [review]
Wait jk rebase onto dev

**Comment by sbrugel** [comment]
bruh

**Review by bladetyr — APPROVED** [review]

**Review by renross — APPROVED** [review]

**Pull request merged**